README.md

```diff
@@ -1,5 +1,7 @@
 # The Lift Web Framework
 
+[![Build Status](https://travis-ci.org/lift/framework.svg?branch=master)](https://travis-ci.org/lift/framework)
```

Comment:
We [threw around this idea a while ago](https://groups.google.com/d/msg/liftweb/hHzK9lgXyKs/6ouEvww0LfwJ) and weren't toooo thrilled at the potential for a misfailed build to be sitting on master.

That said, I think the work @farmdawgnation did has made a passing build an overwhelming likelihood as long as everything is working as it should, so I'm down for adding this now.